src/components/Facility/AssetCreate.tsx

```diff
@@ -55,17 +57,35 @@ const formErrorKeys = [
   "support_email",
   "manufacturer",
   "warranty_amc_end_of_validity",
-  "last_serviced_on",
+  "last_serviced",
   "notes",
 ];
 
-const initError = formErrorKeys.reduce(
-  (acc: { [key: string]: string }, key) => {
-    acc[key] = "";
-    return acc;
-  },
-  {},
-);
+interface AssetT {
```

Comment:
Make sure to use names that make sense; AssetT doesn't tell us what it is.

Let's change it to AssetData instead.